### PR TITLE
[ZEPPELIN-5015]. upgrade shiro

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Simple PR which upgrade shiro to 1.6 because the current 1.4 has some security issues.

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5015

### How should this be tested?
* CI pass and manually tested

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by Reamer** [comment]
LGTM
manually tested. Can you share the travis-ci link.

File: pom.xml [hunk 1]
```diff
@@ -136,7 +136,7 @@
     <commons.collections.version>3.2.2</commons.collections.version>
     <commons.logging.version>1.1.1</commons.logging.version>
     <commons.cli.version>1.4</commons.cli.version>
-    <shiro.version>1.4.2</shiro.version>
```
Comment (Reamer) [inline pom.xml:139]: LGTM

**Review by Reamer — APPROVED** [review]

**Comment by Reamer** [comment]
@zjffdu Can we merge this?

**Comment by zjffdu** [comment]
@Reamer One CI test is still failed

**Review by prabhjyotsingh — APPROVED** [review]
LGTM

**Comment by Reamer** [comment]
Closed with #3976

**Pull request closed without merging**